modules/gapi/include/opencv2/gapi/gmat.hpp

```diff
@@ -213,15 +206,15 @@ struct GAPI_EXPORTS GMatDesc
 static inline GMatDesc empty_gmat_desc() { return GMatDesc{-1,-1,{-1,-1}}; }
 
 #if !defined(GAPI_STANDALONE)
-class Mat;
-GAPI_EXPORTS GMatDesc descr_of(const cv::Mat &mat);
 GAPI_EXPORTS GMatDesc descr_of(const cv::UMat &mat);
 #endif // !defined(GAPI_STANDALONE)
 
+GAPI_EXPORTS GMatDesc descr_of(const cv::Mat &mat);
 /** @} */
 
 // FIXME: WHY??? WHY it is under different namespace?
 namespace gapi { namespace own {
+    class Mat;
```

Comment:
@dmatveev  It is` own::Mat`. `cv::gapi::own::descr_of(...)` needs it. `cv::gapi::own::descr_of(...)` is used in GMatDesc's tests with `own::Mat`. Does this function (cv::gapi::own) make sense?
Same problem with `canDescribe(...)` (variant with own::Mat deleted now, but it cause for removal `CanDescribeOwnMat `test ).
And i see two ways now:
- Remove cv::gapi::own::descr_of(...) and change tests;
- Wrap function canDescribe(...) in cv::gapi::own.